README.md

```diff
@@ -1,2 +1,59 @@
-# JS-TO-DO
-This repository contains source of a To-Do List developed in JavaScript.
+# TO-DO LIST
+
+> This repository contains source of a To-Do List.
+ User can add projects.
+ User can add tasks to projects.
+ User can edit a task or delete it.
+
+![screenshot](./src/screenshot.png)
+
+## Set Up Project Locally
+
+**To get this project set up on your local machine, follow these simple steps:**
+
+**Step 1**<br>
+Navigate through the local folder where you want to clone the repository and run<br>
+`git@github.com:SajjadAhmad14/JS-TO-DO.git` to clone the repo to your local folder.<br>
+or with https<br>
+`https://github.com/SajjadAhmad14/JS-TO-DO.git`.<br>
+**Step 2**<br>
+Run `cd JS-TO-DO`<br>
+**Step 2**<br>
+Run `npm install`<br>
+
+## Built With
+
+- Javascript
+- Webpack
+- HTML
+- CSS
+- Font Awsome 
+- VScode
+
+## Authors
+
+👤 **Jyoti Rai**
+
+- GitHub: [@githubhandle](https://github.com/jrai0792)
+- LinkedIn: [@linkedin](https://www.linkedin.com/in/rai-jyoti)
+- Twitter: [@twitter](https://twitter.com/jyotirai0792)
+
+👤 **Sajjad Ahmad**
+
+- GitHub: [@githubhandle](https://github.com/SajjadAhmad14)
+- Twitter: [@twitterhandle](https://twitter.com/Sajjad_Ahmad14)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/sajjadahmad14)
```

Comment:
## Improve social links
- [ ] Let's replace these placeholder social handles with the actual social handles of our accounts.
```suggestion
- GitHub: [@jrai0792](https://github.com/jrai0792)
- LinkedIn: [Jyoti Rai](https://www.linkedin.com/in/rai-jyoti)
- Twitter: [@jyotirai0792](https://twitter.com/jyotirai0792)

👤 **Sajjad Ahmad**

- GitHub: [@SajjadAhmad14](https://github.com/SajjadAhmad14)
- Twitter: [@Sajjad_Ahmad14](https://twitter.com/Sajjad_Ahmad14)
- LinkedIn: [Sajjad Ahmad](https://www.linkedin.com/in/sajjadahmad14)
```